posthog/models/team.py

```diff
@@ -106,18 +105,19 @@ class Team(models.Model):
     completed_snippet_onboarding: models.BooleanField = models.BooleanField(default=False)
     ingested_event: models.BooleanField = models.BooleanField(default=False)
     uuid: models.UUIDField = models.UUIDField(default=UUIDT, editable=False, unique=True)
+    session_recording_opt_in: models.BooleanField = models.BooleanField(default=False)
```

Comment:
This seems completely unrelated to this PR and the UX for this is still in open discussion, I would remove for now.

Reply:
Ah, this is actually not new at all, I just moved this line to be above the deprecated fields (as there are new such ones).